test/test_iterdatapipe.py

```diff
@@ -703,6 +703,28 @@ def _custom_len_fn(token):
         self.assertEqual(res_before_reset, exp_before_reset)
         self.assertEqual(res_after_reset, exp_after_reset)
 
+        # Functional test: Padded tokens exceeding max_token_count
+        source_data = ["111", "1111", "11111"]  # 3, 4, 5
+        source_dp = IterableWrapper(source_data)
+        batch_dp = source_dp.max_token_bucketize(max_token_count=7)
+        exp_batch = [["111", "1111"], ["11111"]]
+
+        self.assertEqual(list(batch_dp), exp_batch)
+
+        # Functional test: Padded tokens not exceeding max_token_count
+        source_data = ["111", "1111", "11111"]  # 3, 4, 5
+        source_dp = IterableWrapper(source_data)
+        batch_dp = source_dp.max_token_bucketize(max_token_count=7, count_padding=True)
+        exp_batch = [["111"], ["1111"], ["11111"]]
+        self.assertEqual(list(batch_dp), exp_batch)
```

Comment:
Could you please change this test to make sure part of input can be aggregated into a batch but part can not?